packages/renderer/index.js

```diff
@@ -2,12 +2,23 @@
 
 var events = require('events');
 
+var express = require('express');
 var mocks = require('node-mocks-http');
 
 var createMiddleware = require('hops-middleware');
 
-module.exports = function createRenderer (webpackConfig, watchOptions) {
-  var middleware = createMiddleware(webpackConfig, watchOptions);
+module.exports = function createRenderer (options) {
+  var hopsConfig = options.hopsConfig || {};
+  var router = new express.Router();
+
+  if (hopsConfig.bootstrapServer) {
+    hopsConfig.bootstrapServer(router, hopsConfig);
+  }
+
+  router.use(createMiddleware(
+    options.webpackConfig,
+    options.watchOptions
+  ));
 
   return function (location) {
```

Comment:
I think we need to be able to pass more than location here to configure the request.
For example, in order to be able to change the language, we need to set headers (cookie or accept-language)